src/script/squirrel.cpp

```diff
@@ -764,6 +768,12 @@ void Squirrel::Uninitialize()
 {
 	ScriptAllocatorScope alloc_scope(this);
 
+	/* Remove the delegation */
+	sq_pushroottable(this->vm);
+	sq_pushnull(this->vm);
+	sq_setdelegate(this->vm, -2);
+	sq_pop(this->vm, 1);
+
 	/* Clean up the stuff */
 	sq_pop(this->vm, 1);
```

Comment:
I'm not sure this works as expected when uninitialisation is done while the script is running (like AI reset).
It will pop stack top, but the intention was to clear the stack I think.